diff --git a/src/CodeAnalysis/CodeAnalysis.csproj b/src/CodeAnalysis/CodeAnalysis.csproj
index f101c70..5493882 100644
--- a/src/CodeAnalysis/CodeAnalysis.csproj
+++ b/src/CodeAnalysis/CodeAnalysis.csproj
@@ -14,7 +14,6 @@
     <PackageReference Include="ThisAssembly.AssemblyInfo" Version="1.1.3" PrivateAssets="all" />
     <PackageReference Include="ThisAssembly.Strings" Version="1.1.3" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" Pack="false" />
-    <PackageReference Include="Devlooped.SponsorLink" Version="0.10.2" />
   </ItemGroup>
 
   <ItemGroup>
diff --git a/src/CodeAnalysis/SponsorLinker.cs b/src/CodeAnalysis/SponsorLinker.cs
deleted file mode 100644
index 3e4618a..0000000
--- a/src/CodeAnalysis/SponsorLinker.cs
+++ /dev/null
@@ -1,21 +0,0 @@
-﻿using System;
-using Devlooped;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Diagnostics;
-
-namespace NuGetizer;
-
-[DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic, LanguageNames.FSharp)]
-class SponsorLinker : SponsorLink
-{
-    public SponsorLinker() : base(SponsorLinkSettings.Create(
-        "devlooped", "NuGetizer",
-        version: new Version(ThisAssembly.Info.Version).ToString(3),
-        // Add an extra digit so the SL diagnostics have the same length as ours
-        diagnosticsIdPrefix: "NG1"
-#if DEBUG
-        , quietDays: 0
-#endif
-        ))
-    { }
-}
\ No newline at end of file
diff --git a/src/Directory.Build.targets b/src/Directory.Build.targets
index 708c456..a81ebd7 100644
--- a/src/Directory.Build.targets
+++ b/src/Directory.Build.targets
@@ -25,6 +25,12 @@
     <IsPackable Condition="'$(PackageId)' != ''">true</IsPackable>
   </PropertyGroup>
   
+  <PropertyGroup>
+    <!-- Allow easily changing the prefix of the package names -->
+    <PackageId Condition="'$(BuildPackageBaseName)' != ''">$([System.String]::Copy('$(PackageId)').Replace('NuGetizer', '$(BuildPackageBaseName)'))</PackageId>
+    <PackageId Condition="'$(BuildPackageBaseName2)' != ''">$([System.String]::Copy('$(PackageId)').Replace('nugetize', '$(BuildPackageBaseName2)'))</PackageId>
+  </PropertyGroup>
+
   <ItemGroup Condition="'$(IsPackable)' == 'true'" Label="NuGet">
     <!-- This is compatible with nugetizer and SDK pack -->
 
diff --git a/src/NuGetizer.Tasks/NuGetizer.Tasks.csproj b/src/NuGetizer.Tasks/NuGetizer.Tasks.csproj
index 55d70bb..c9abafe 100644
--- a/src/NuGetizer.Tasks/NuGetizer.Tasks.csproj
+++ b/src/NuGetizer.Tasks/NuGetizer.Tasks.csproj
@@ -41,6 +41,13 @@
     <None Update="NuGetizer.Tasks.targets" Pack="false" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BuildPackageBaseName)' != ''">
+    <None Update="UnNuGetizer.props" PackagePath="build\UnNuGetizer.props" />
+    <None Update="UnNuGetizer.targets" PackagePath="build\UnNuGetizer.targets" />
+    <None Update="UnNuGetizer.MultiTargeting.props" PackagePath="buildMultiTargeting\UnNuGetizer.props" />
+    <None Update="UnNuGetizer.MultiTargeting.targets" PackagePath="buildMultiTargeting\UnNuGetizer.targets" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\CodeAnalysis\CodeAnalysis.csproj" ReferenceOutputAssembly="false" />
   </ItemGroup>
diff --git a/src/NuGetizer.Tasks/UnNuGetizer.MultiTargeting.props b/src/NuGetizer.Tasks/UnNuGetizer.MultiTargeting.props
new file mode 100644
index 0000000..2d46452
--- /dev/null
+++ b/src/NuGetizer.Tasks/UnNuGetizer.MultiTargeting.props
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="NuGetizer.props" />
+</Project>
\ No newline at end of file
diff --git a/src/NuGetizer.Tasks/UnNuGetizer.MultiTargeting.targets b/src/NuGetizer.Tasks/UnNuGetizer.MultiTargeting.targets
new file mode 100644
index 0000000..fea8e3e
--- /dev/null
+++ b/src/NuGetizer.Tasks/UnNuGetizer.MultiTargeting.targets
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="NuGetizer.targets" />
+</Project>
\ No newline at end of file
diff --git a/src/NuGetizer.Tasks/UnNuGetizer.props b/src/NuGetizer.Tasks/UnNuGetizer.props
new file mode 100644
index 0000000..2d46452
--- /dev/null
+++ b/src/NuGetizer.Tasks/UnNuGetizer.props
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="NuGetizer.props" />
+</Project>
\ No newline at end of file
diff --git a/src/NuGetizer.Tasks/UnNuGetizer.targets b/src/NuGetizer.Tasks/UnNuGetizer.targets
new file mode 100644
index 0000000..fea8e3e
--- /dev/null
+++ b/src/NuGetizer.Tasks/UnNuGetizer.targets
@@ -0,0 +1,3 @@
+<Project>
+  <Import Project="NuGetizer.targets" />
+</Project>
\ No newline at end of file
diff --git a/src/dotnet-nugetize/Program.cs b/src/dotnet-nugetize/Program.cs
index e61337d..d025bfa 100644
--- a/src/dotnet-nugetize/Program.cs
+++ b/src/dotnet-nugetize/Program.cs
@@ -7,9 +7,7 @@ using System.Linq;
 using System.Reflection;
 using System.Text;
 using System.Threading;
-using System.Threading.Tasks;
 using System.Xml.Linq;
-using Devlooped;
 using Mono.Options;
 using Spectre.Console;
 using Spectre.Console.Rendering;
@@ -29,45 +27,10 @@ class Program
     string items;
     List<string> extra;
 
-    static async Task<int> Main(string[] args)
+    static int Main(string[] args)
     {
-        var status = SponsorCheck.CheckAsync(Directory.GetCurrentDirectory(), "devlooped", "NuGetizer", "dotnet-nugetize", ThisAssembly.Project.Version);
         var result = new Program().Run(args);
 
-        // No need to check sponsorlink status if we couldn't render useful results.
-        if (result == 0)
-        {
-            var value = await status;
-            if (value == null)
-                return result;
-
-            switch (value.Value)
-            {
-                case SponsorStatus.AppMissing:
-                    Warning(
-                        AppMissing.Header,
-                        new Markup(AppMissing.Message1("NuGetizer", "devlooped")),
-                        new Grid().AddColumns(2)
-                            .AddRow(
-                                new Markup(AppMissing.Message2),
-                                new Text("https://github.com/apps/sponsorlink",
-                                    new Style(Color.Blue, decoration: Decoration.Underline, link: "https://github.com/apps/sponsorlink"))));
-                    break;
-                case SponsorStatus.NotSponsoring:
-                    Warning(
-                        NotSponsoring.Header,
-                        new Markup(NotSponsoring.Message("NuGetizer")),
-                        new Text("https://github.com/sponsors/devlooped",
-                            new Style(Color.Blue, decoration: Decoration.Underline, link: "https://github.com/apps/sponsorlink")));
-                    break;
-                case SponsorStatus.Sponsoring:
-                    AnsiConsole.Write(new Markup($":heart_decoration: [grey30]{Sponsoring.Message("NuGetizer", "devlooped")}[/]"));
-                    break;
-                default:
-                    break;
-            }
-        }
-
         return result;
     }
 
diff --git a/src/dotnet-nugetize/dotnet-nugetize.csproj b/src/dotnet-nugetize/dotnet-nugetize.csproj
index c0ae27a..3a024c8 100644
--- a/src/dotnet-nugetize/dotnet-nugetize.csproj
+++ b/src/dotnet-nugetize/dotnet-nugetize.csproj
@@ -30,8 +30,6 @@
     <PackageReference Include="Spectre.Console.Analyzer" Version="0.46.0" PrivateAssets="all" />
     <PackageReference Include="ThisAssembly.Project" Version="1.2.9" PrivateAssets="all" />
     <PackageReference Include="ThisAssembly.Strings" Version="1.2.9" PrivateAssets="all" />
-    <PackageReference Include="Devlooped.SponsorLink" Version="0.9.9" />
-    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="7.0.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" />
   </ItemGroup>
 
